alws/dramatiq/build.py

```diff
@@ -63,7 +71,11 @@ def start_build(build_id: int, build_request: Dict[str, Any]):
 
 @dramatiq.actor(
     max_retries=0,
-    priority=1
+    priority=1,
+    time_limit=3600000,
```

Comment:
Pls, add comment for this value from your latest changes for dramatiq

Reply:
Yeah, probably need to move it to constants